### DISCUSSION TITLE
Are you OK with Dark Reader being a paid app?

### DISCUSSION THREAD

**Discussion by Hubro** [darkreader/darkreader#9297] [Polls]
I was hyped to realize today that Dark Reader has been released for iOS and ipadOS. I promptly went to install it, and my hype immediately died when I realized they are charging for it.

It was my understanding that Dark Reader is an open source, donation-supported project, which is why I've been donating to it for years. Now, apparently, they are selling it for a profit? ~~And worse yet, *I have to buy it too!?* After I've already donated well over $100 to this project?~~

I've searched around for an explanation but I've found nothing. I suppose they could just be trying to cover the costs associated with publishing an app on the App Store, but it seems priced **way** too high for that.

Surprisingly, I've found nobody complaining about this. Am I the only one feeling scammed here? Am I missing something? Is everybody else really OK with this?

---

EDIT: I contacted Dark Reader over E-mail and asked about this, and they sent me a promo code to install Dark Reader for free. So it turns out, long term supporters don't actually have to purchase the app as long as you contact the official E-mail. I think this could be better communicated, especially on the Open Collective page. In any case, I have much less of a problem with this now.

**Comment by alexanderby** [comment]
As mentioned in my email, only 1 of 2000 users, 0.05% support us financially. For some reason people tend to purchase an item rather than making a donation. We have run a poll recently, many people stated the amounts they can donate, but very few did.

The paid macOS and iOS version was the way for this project to exist and be improved so far during these 4 years. As I lived in a poor country till last year, the income was enough. I remember seeing around 10-20 complaints about macOS version being paid, it's 0.0003%. I can't remember a complaint from our backers. We could highlight it somewhere, if only it would not break the App Store policies. They don't allow sending promo codes in exchange for items outside of the App Store.

I see nothing criminal or shameful if developers are paid for their work. Going further, soon I may join a company, developing an open source project, and the proposed salary is much higher than the whole Dark Reader income with its 3 team members at the moment.

Thanks for your support. I hope you don't find this to be scam anymore. I will think on highlighting the issue in a way that doesn't break the App Store policies.

**Reply by lambdakilo** [reply]
Stack Overflow doesn't seem to have any objections against moving from paid to free: https://stackoverflow.com/questions/7237807/can-i-change-my-app-from-free-to-paid-or-from-paid-to-free-any-time
Any thoughts on this kind of transition?

**Comment by dftf-stu** [comment]
Hi.  After you pay for this app and get the e-mail from "Paddle.com Market Limited", is there a setting somewhere in the extension to enter an e-mail address or something?

When I click on the extension, it still says "Pay for using Dark Reader".  So, after a user has, how do we then make this message go-away?

**Comment by alexanderby** [comment]
Hi @dftf-stu, there's a way on Edge browser at the moment. We will implement it for other browsers soon. Please ignore that message in the meantime.

**Reply by rubaboo** [reply]
Hi, this is not a criticism, I agree everyone enjoying a product for a while should ideally pay for it. However, there are some people to whom the biggest incentive for being a paid customer is not getting nagged about it. Please implement soon 👍 Thank you.

**Reply by Parsa307** [reply]
> Hi @dftf-stu, there's a way on Edge browser at the moment. We will implement it for other browsers soon. Please ignore that message in the meantime.

Please don't! I understand that making it paid might be a quick profit method, But let all the user's enjoy all features. Make it optional for donation!

**Reply by dftf-stu** [reply]
@rubaboo I did actually raise ticket #13944 about this, as I felt it was odd having it tucked-away as a discussion only, where fewer people would ever see it

**Reply by dftf-stu** [reply]
@Parsa307: no-one here is suggesting to make the extension payment-only; donation should remain optional, but the message within the extension asking people to consider donating those of us who _have_ paid should be able to then remove.  That's literally the only one thing being asked for here, not other features to become paywalled

**Comment by dftf-stu** [comment]
@alexanderby: any further thoughts on this?

Does seem pointless to pay if we cannot even get rid of the nag in the extension...

![DarkReaderRemovePayNag](https://github.com/user-attachments/assets/68d507fb-f65f-4c33-864a-7927dc28a079)

**Comment by jimmy-zhening-luo** [comment]
Hey, I am pretty sure I've paid for this app more than once (including just now), and I still get a large, annoying, noisy banner telling me to pay for the app. Can you please fix this asap?

**Reply by dftf-stu** [reply]
Yeah, as someone else who has paid it's crazy @alexanderby and the team can't just add something into the app where you could enter an e-mail address and then a code they send you to make this message go-away.

There are literally _zero_ other benefits to paying for the app, given no features are paywall-locked, nor do you get priority support or priority preference for feature-requests, or similar.  So removing the nag prompt would be one simple thing they could surely offer?

Can't say it's surprising "only 1 of [every] 2000 users" offer to donate when this simple thing gets ignored for so-long 🤷🏻‍♂️

**Reply by dftf-stu** [reply]
(Forgot ticket #13944 is actually open for this)

**Reply by alexanderby** [reply]
Sorry guys, the work on the new user interface and the activation pipeline takes more time than expected.

I will implement the rough method of hiding the message in the old UI shortly. Just to comfirm, which browsers do you use?

**Reply by dftf-stu** [reply]
@alexanderby: mostly Firefox and Microsoft Edge when on desktop.

Brave mostly on mobile (though I'm not aware if Dark Reader is available on mobile platforms, outside of Firefox -- but most mobile browsers offer a built-in dark-mode toggle on sites anyway, which mostly gets the job done).

(Have to add: it would be great if Firefox on desktop could stop with the "Dark Reader is slowing-down your browser" message.  I'm aware this isn't strictly an issue on your-side, but still!)
